internal/app/user_login.go

```diff
@@ -0,0 +1,61 @@
+package app
+
+import (
+	"encoding/json"
+	"github.com/poggerr/gophermart/internal/authorization"
+	"github.com/poggerr/gophermart/internal/logger"
+	"github.com/poggerr/gophermart/internal/models"
+	"io"
+	"net/http"
+	"time"
+)
+
+func (a *App) UserLogin(res http.ResponseWriter, req *http.Request) {
+	body, err := io.ReadAll(req.Body)
+	if err != nil {
+		a.sugaredLogger.Info(err)
+		res.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	var user models.User
+
+	err = json.Unmarshal(body, &user)
+	if err != nil {
+		a.sugaredLogger.Info(err)
+		res.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	userID, isVerify := a.strg.TakeUserID(user.Username)
```

Comment:
```suggestion
	userID, isVerify := a.strg.UserID(user.Username)
```
итак понятно что получаем, префиксы использовать не нужно